<h1>pandas.errors.IncompatibilityWarning</h1> <dl class="py exception"> <dt class="sig sig-object py" id="pandas.errors.IncompatibilityWarning"> <em class="property"><span class="pre">exception</span></em><span class="sig-prename descclassname"><span class="pre">pandas.errors.</span></span><span class="sig-name descname"><span class="pre">IncompatibilityWarning</span></span><a class="reference external" href="https://github.com/pandas-dev/pandas/blob/v2.2.2/pandas/errors/__init__.py#L687-L690"><span class="viewcode-link"><span class="pre">[source]</span></span></a>
</dt> <dd>
<p>Warning raised when trying to use where criteria on an incompatible HDF5 file.</p> </dd>
</dl><div class="_attribution">
  <p class="_attribution-p">
    &copy; 2008&ndash;2022, AQR Capital Management, LLC, Lambda Foundry, Inc. and PyData Development Team<br>Licensed under the 3-clause BSD License.<br>
    <a href="https://pandas.pydata.org/pandas-docs/version/2.2.2/reference/api/pandas.errors.IncompatibilityWarning.html" class="_attribution-link">https://pandas.pydata.org/pandas-docs/version/2.2.2/reference/api/pandas.errors.IncompatibilityWarning.html</a>
  </p>
</div>
